pkg/kpipelines/syntax/pipeline.go

```diff
@@ -4,23 +4,25 @@ import (
 	"crypto/rand"
 	"encoding/hex"
 	"fmt"
-	"github.com/jenkins-x/jx/pkg/kpipelines"
+	"io"
+	"io/ioutil"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jenkins-x/jx/pkg/log"
 	pipelinev1alpha1 "github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
 	"github.com/knative/pkg/apis"
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v2"
-	"io"
-	"io/ioutil"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/equality"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"regexp"
-	"strings"
-	"time"
 )
 
 const (
-	PipelineFileName = "jenkins-x.yaml"
+	defaultContainerName = "maven"
```

Comment:
Just a temp thing? Not sure what would make the most sense here, but I don't know if JX has the same Java-leaning users as regular Jenkins.